lib/registry/src/config.rs

```diff
@@ -4,6 +4,8 @@ use serde::Serialize;
 use std::collections::BTreeMap;
 use std::path::{Path, PathBuf};
 
+pub static GLOBAL_CONFIG_DATABASE_FILE_NAME: &str = "wapm.sqlite";
+
 #[derive(Deserialize, Default, Serialize, Debug, PartialEq, Eq)]
 pub struct PartialWapmConfig {
```

Comment:
This should be named Wasmer config isn't it?

Reply:
I mean yeah if you want to break compatibility with wapm (settings would not be shared between wapm and wasmer)

Reply:
What do you mean? `PartialWapmConfig` can be called `PartialWasmerConfig`.

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3432/commits/c29377f298aa017144658446db4530d7d5d1ec66